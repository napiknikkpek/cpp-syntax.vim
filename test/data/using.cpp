using namespace boost;
using namespace boost::hana;
using T;
using boost::hana::fold;
using T = My_item;
using T = std::vector<int>;
using T = std::index<3>;
using T = typename Base::type;
using T = decltype(func(a));
using T = typename decltype(a)::type;
