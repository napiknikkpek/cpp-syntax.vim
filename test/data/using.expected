aaaaa.aaaaaaaaa.ttttt.
aaaaa.aaaaaaaaa.ttttt..tttt.
aaaaa.t.
aaaaa.ttttt..tttt..tttt.
aaaaa.t...ttttttt.
aaaaa.t...ttt..tttttt.zzz..
aaaaa.t...ttt..ttttt.o..
aaaaa.t...aaaaaaaa.tttt..tttt.
aaaaa.t...aaaaaaaa.ffff.x...
aaaaa.t...aaaaaaaa.aaaaaaaa.x...tttt.
